en/02_Developer_Guides/03_Forms/01_Validation.md

```diff
@@ -116,8 +115,42 @@ You can also override the entire `Form` validation by subclassing `Form` and def
 
 ## Form action validation
 
+### `ValidationException`
+
+At times it's not possible for all validation or recoverable errors to be pre-determined in advance of form
+submission, such as those generated by the form [Validator](api:SilverStripe\Forms\Validator) object. Sometimes errors may occur within form
```

Comment:
```suggestion
submission, such as those generated by the form [Validator](api:SilverStripe\Forms\Validator). Sometimes errors may occur within form
```

Reply:
Done. Also removed "object" in a couple other places where it was used in a similar context.